### PR TITLE
imagemagick is a prerequisite for adding Spree to existing Rails application

### PR DESCRIPTION
**Pull request by alanzheng88**
I ran into this issue when installing spree for the first time and I thought this may help others resolve this issue.

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://alanzheng88-master.spree-spree-9ed724b77adec04c49c2.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://alanzheng88-master.spree-spree-9ed724b77adec04c49c2.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: README.md [hunk 1]
```diff
@@ -54,6 +54,10 @@ spree install my_store
 This will add the Spree gem to your Gemfile, create initializers, copy migrations
 and optionally generate sample products and orders.
 
+If you get an "sh: identify: command not found" error then you can try installing magick.
```
Comment (JDutil) [inline README.md]: Can you update this to say `try installing imagemagick.`

Comment (alanzheng88) [inline README.md]: Yep done.

**Comment by JDutil** [comment]
Thanks

**Pull request merged**